packages/react-search-ui/src/containers/Sorting.js

```diff
@@ -2,24 +2,45 @@ import PropTypes from "prop-types";
 import { Component } from "react";
 import { withSearch } from "..";
 import { Sorting } from "@elastic/react-search-ui-views";
+import SortList from "../types/SortList";
 
 import { SortOption } from "../types";
 
-function findSortOption(sortOptions, sortString) {
-  const [value, direction] = sortString.split("|||");
-  return sortOptions.find(
-    option => option.value === value && option.direction === direction
-  );
+function findSortOption(sortOptions, sortData) {
+  let value;
+  let direction;
+  if (Array.isArray(sortData)) {
+    value = sortData;
+
+    // We don't care about option.direction in this case since direction is part of the value array.
+    return sortOptions.find(
+      option => JSON.stringify(option.value) === JSON.stringify(value)
```

Comment:
Can you explain the reasoning behind stringifying these objects and then comparing the strings?

Reply:
It didn't work without JSON.stringify. The find method was returning undefined until I added the JSON.stringify.